### PR TITLE
Hacky freon fixes pending the rework

### PR DESCRIPTION
**Pull request by Iamgoofball**
fixes the slipping teleport

more changes are pending the rewrite since they're gameplay balance but this is a fix for the slip shit

### CODE REVIEW

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -119,8 +119,10 @@
 					step(C, olddir)
 					C.spin(1,1)
 		else if(lube&SLIDE_ICE)
+			C.slipping = TRUE
 			spawn(1)
 				step(C, olddir)
+			C.slipping = FALSE
```
Comment (ChangelingRain) [inline code/game/turfs/open.dm]: this literally should not work
you are spawning off the step() so, accordingly, you are setting this to TRUE and then just fucking setting it back to FALSE

**Comment by kevinz000** [comment]
MAKE WATCHER FREEZE BEAMS AND SPACE NOT FREEZE YOU JESUS.

**Comment by Iamgoofball** [comment]
@kevinz000 balance changes, those are in the freon rework/Goofmos PR

**Comment by optimumtact** [comment]
Which will never be merged thanks to it's inclusion of a reagent gas

**Comment by Shadowlight213** [comment]
@Iamgoofball the watcher freezing doesn't seem to ever unfreeze you. I'm getting ahelps about miners being permafrozen. That sounds like a bug to me.

**Comment by Iamgoofball** [comment]
that's weird, I'll check it out

**Comment by Shadowlight213** [comment]
Also do you want me to make a pr to fix the air alarm interfaces? The ui data for freon scrubbers seems to not work.

**Pull request merged**